pkg/objectstate/objectstate.go

```diff
@@ -37,3 +37,7 @@ func (obst ObjectState) IsNotFoundError() bool {
 	}
 	return apierrors.IsNotFound(obst.Error)
 }
+
+func (obst ObjectState) IsCreateOrUpdate() bool {
```

Comment:
needs test also to catch the `non-nil interface point-to-nil` (https://trstringer.com/go-nil-interface-and-interface-with-nil-concrete-value/)

Reply:
solved in the last upload